Bump python + django

therefromhere opened the pull request:
- Added django 1.11, 2.0 and Python 3.6
- bump two scoops to 1.11

ssteinerx commented:
The only hesitation I have about this is that some of the changes are to the *generated* package and, as far as I know, there are currently no tests in this project that actually generate a package and run the tests on that  package.

Is that correct?

kakulukia commented:
@therefromhere please answer, because i just wanted to report an issue for that only to see its already kinda fixed :)

therefromhere commented:
Sorry, I'd need to check, I'm not sure if the generated package is tested currently.

therefromhere commented:
@ssteinerx / @kakulukia  there is a test that attempts to run the tests of the generated project https://github.com/pydanny/cookiecutter-djangopackage/blob/e9c706b2b743bc8ef925827abcc1c6cda2bbb056/tests/test_bake_project.py#L278 - but I'm not convinced this working correctly (eg it doesn't fail for me when I edit the stub `test_something` here to fail https://github.com/pydanny/cookiecutter-djangopackage/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/tests/test_models.py#L21 )

therefromhere commented:
I've added #306 as a suggested fix for this.

browniebroke commented on {{cookiecutter.repo_name}}/tox.ini:
```diff
@@ -1,9 +1,11 @@
-    {py27,py34,py35}-django-master{% endif %}
+    {py27,py33,py34,py35,py36}-django-18{% endif %}{% if '1.9' in cookiecutter.django_versions %}
+    {py27,py34,py35,py36}-django-19{% endif %}{% if '1.10' in cookiecutter.django_versions %}
+    {py27,py34,py35,py36}-django-110{% endif %}{% if '1.11' in cookiecutter.django_versions %}
```
Although that is probably working, Python 3.6 [isn't listed in the versions](https://docs.djangoproject.com/en/2.0/faq/install/#what-python-version-can-i-use-with-django) usable with Django < 1.11.

therefromhere replied:
I've removed that now

browniebroke commented:
I've generated [a project](https://github.com/browniebroke/dj-package) from that branch and [set up a Travis build](https://travis-ci.org/browniebroke/dj-package).

browniebroke commented on {{cookiecutter.repo_name}}/.travis.yml:
```diff
@@ -3,29 +3,39 @@
+  - TOX_ENV=py36-django-18
   - TOX_ENV=py35-django-18
   - TOX_ENV=py34-django-18
   - TOX_ENV=py33-django-18
```
Given that [3.3 has reached EOL](https://devguide.python.org/#status-of-python-branches), I suppose it could be dropped?

pydanny replied:
Yes, we should drop 3.3. In fact, I argue we should drop everything but 3.6

kakulukia replied:
tho 3.5 still is the most up to date version u can install (with the default package manager) in some linux distros .. i also think, that pushing everyone to install 3.6 is worth it! 
f-expressions-❤

therefromhere replied:
By the same argument would people suggest should we also drop tox/travis checks on django 1.9, 1.10?   (and 1.8 once it goes out of LTS support in April?)

therefromhere replied:
I've dropped py33.

I guess we can drop Django<1.11 in a week or so after 1.8 goes out of LTS.

therefromhere commented:
It looks like @pydanny already made a lot of these changes in master, I've updated the branch to fix a few issues (eg py27 isn't valid with Django 2, fixed the tests).
